style.css

```diff
@@ -77,6 +50,62 @@ body {
     
 }
 
+.menu {
+    display: flex;
+    flex: 1; 
+    flex-direction: column;
+}
+
+#title-box-wrapper {
+    width: 431px;
+    height: 28px;
+    background-color: black;
+    margin:auto;
+    /* margin-left: 370px;  */
+    order: 2;
+}
+
+#title-box {
+    text-align: left;  
+    font-size: 16px;
+    font-family: 'Helvetica Neue';
+    color: lightgray;
+}
+
+#buttons {
+    width: 431px;
+    height: 18 px;
+    margin: auto;
+    background-color: white;
+    order: 3;
+    display: flex;
+    flex-direction: column;
+}
+
+#start-button {
+    /* margin-left: 370px; */
+    display: inline-block;
+}
+
+#stop-button {
+    /* margin-left: 10px; */
+    display: inline-block;
+}
+
+.vol-slide-container {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    order: 4;
```

Comment:
`order`, nice 👌  -  but it's usually preferable to define the order by ordering the html elements themselves rather than in css -- it makes it more difficult for a person to read the html and know what will appear where. I'd recommend only using order in sticky situations where you can't do it with html alone.

Reply:
That's what I tried to do, but I thought I had to do it this way in order to use flexbox, column (to get to make a column/match the original beepbox), because otherwise the divs go horizontally...I'm not sure if there's another easier solution for this

Reply:
ah `flex-direction: column` will make the divs appear in a column, but maintain whatever order is in the html